lua/graph.lua

```diff
@@ -273,14 +273,17 @@ truck = {
 ["discouraged"] = "false",
 ["permissive"] = "true",
 ["unsuitable"] = "false",
-["agricultural;forestry"] = "false",
 ["official"] = "true",
 ["forestry"] = "false",
-["destination;delivery"] = "true",
 ["permit"] = "true",
 ["residents"] = "true"
 }
 
+truck_hgv = {
+  ["designated"] = "true",
+  ["local"] = "true",
```

Comment:
the meaning of local doesn't seem well defined, and possibly means "destination only" rather than a designated way: https://wiki.openstreetmap.org/wiki/Key:hgv Or am I missing something?